installer/install.sh

```diff
@@ -1,9 +1,12 @@
-#!/bin/bash -ex
+#!/bin/bash -e
```

Comment:
Probably better for production use; we can always set -x explicitly before running.